temfpy/interpolation.py

```diff
@@ -1,5 +1,5 @@
 """Test capabilities for interpolation.
-This module contains a host of models and functions often used for testing
+This module contains a **host of models** and functions often used for testing

```

Comment:
It says a 'host of models and functions,' but there is only one function in the section. Will you add more to this page at some point or would it be better to rephrase?